demos/stable-diffusion-1.5/index.js

```diff
@@ -986,6 +987,10 @@ async function executeStableDiffusion() {
 
     const halfLatentElementCount = latentsTensor.size / 2; // Given [2, 4, 64, 64], we want only the first batch.
     let latents = await latentsTensor.getData();
+    if (isFloat16ArrayAvailable) {
+        // If Float16Array is available, convert latents back to Uint16Array for post-processing.
+        latents = new Uint16Array(latents.buffer, latents.byteOffset, latents.length);
```

Comment:
```
if (isFloat16ArrayAvailable) { ...
    new Uint16Array(...)
}
```

Backwards? If you have the new Float16Array, why convert it to old Uint16 Array? Is there still older code that expects Uint16Array?

Reply:
Many util functions in this file only accept `Uint16Array` input, e.g. `prescaleLatentSpace`, `denoiseLatentSpace`, `applyVaeScalingFactor` and etc, I just make a quick fix to make the demos work ASAP.

We can do more improvement later.

Reply:
Ok.